myadmin-angular/projects/myadmin-lib/src/lib/services/study-material.service.ts

```diff
@@ -1,8 +1,9 @@
 import {Injectable} from "@angular/core";
 import {HttpClient} from "@angular/common/http";
 import {Observable} from "rxjs";
-import {StudentModuleEnrolmentInfo, StudyMaterialDetailInfo} from "../info-objects";
-import {StudentInfo} from "myadmin-lib";
+import {StudentInfo} from "../info-objects/student-info";
+import {StudentModuleEnrolmentInfo} from "../info-objects/student-module-enrolment-info";
```

Comment:
I can't remember but I thought we agreed to dropping the Student prefix and rename it to be ModuleEnrolmentInfo.